src/js/constants.js

```diff
@@ -2,6 +2,20 @@ const CheckModel = {
     ALL: 'all',
     PARENT: 'parent',
     LEAF: 'leaf',
+    flatKeys: [
+        'label',
+        'value',
+        'children',
+        'parent',
+        'isChild',
+        'isParent',
+        'isLeaf',
+        'showCheckbox',
+        'disabled',
+        'treeDepth',
+        'index',
+        'extraKeys',
```

Comment:
I think we actually should remove this line. Wasn't able to test this locally. Was getting an error on `npm run examples` and `npm run test` had some tests that appeared to be failing for me on the `master` branch w/o my changes.

```suggestion
```